htdocs/GetStatic.php

```diff
@@ -33,7 +33,13 @@
 // Ensures the user is logged in, and parses the config file.
 require_once "NDB_Client.class.inc";
 $client = new NDB_Client();
-if ($client->initialize("../project/config.xml") == false) {
+
+$configFile = getenv('LORIS_DB_CONFIG');
```

Comment:
There's more situations than this to take into account.

I was going to suggest moving https://github.com/aces/Loris/blob/17.1-dev/php/libraries/NDB_Config.class.inc#L71 into a helper function, but now that I'm looking at it.. wouldn't this work if you just changed it to `$client->initialize(null)`?